starboard/android/shared/media_codec_bridge_eradicator.cc

```diff
@@ -47,7 +53,19 @@ struct EradicateParam {
 void MediaCodecBridgeEradicator::WaitForPendingDestructions() {
   ScopedLock scoped_lock(mutex_);
   while (!j_media_codec_bridge_set_.empty()) {
-    condition_variable_.Wait();
+    bool signaled =
+        condition_variable_.WaitTimed(ERADICATOR_THREAD_TIMEOUT_MICROSECOND);
+    if (!signaled) {
+      // condition_variable_ timed out
+      SB_LOG(WARNING)
```

Comment:
Instead of an error log, can we fire a playback error to h5 player? So that, when running experiment, if the last destruction was hang, we can catch that error from qoe data.

Reply:
yes, query this on server will be great. 
Please give me an example of how to do it.

Reply:
Updated